### ISSUE TITLE
Make sure all kernels support containers

### ISSUE THREAD

**Issue by zhuowei** [postmarketOS/pmbootstrap#693]
I noticed that some devices do not explicitly enable CONFIG_NAMESPACE, which is one of the options required for Linux containers. 

There's quite a few projects using containers (Anbox, Docker, etc), so we should investigate which kernel options are required for them, and then check/enable them on all the supported kernels.

**Comment by ollieparanoid** [comment]
Good idea! There have been quite a few people asking about anbox support.
